### PR TITLE
community/wlroots: add elogind support

### PR DESCRIPTION
**Pull request by PureTryOut**
CC @ddevault, could you please review this?

### CODE REVIEW

**Comment by Ikke** [comment]
@maxice8 That can also be done when applied.

**Comment by maxice8** [comment]
**I** **will** **do** **it** **later**

**Comment by ddevault** [comment]
LGTM but I would explicitly enable elogind support with meson options to catch issues earlier

Aside: I think that elogind design-wise is pretty bad and I would sooner await a good replacement than ship it with Alpine. YMMV

**Comment by PureTryOut** [comment]
@ddevault thanks for the review! I explicitely enabled logind support and set the logind provider to elogind. About elogind, it is already shipped with Alpine and at the moment it's the only real option aside from the dead-upstream ConsoleKit2. Do you know of any replacement in the works?

@maxice8 and @Ikke: since I had to make a change anyway, I took the liberty to modernize the APKBUILD as well.

**Comment by ddevault** [comment]
The other option is the status quo: setuid. But I'm not torn up about it

File: community/wlroots/APKBUILD [hunk 1]
```diff
@@ -21,22 +21,21 @@ makedepends="eudev-dev
 	wayland-protocols
 	xcb-util-image-dev
 	xcb-util-wm-dev
-	"
+	elogind-dev"
```
Comment (tcely) [inline community/wlroots/APKBUILD]:
```suggestion
	elogind-dev
	"
```

**Pull request closed without merging**